### ISSUE TITLE
Get Color Editor return null in .net core

### ISSUE THREAD

**Issue by TempeBrennan** [dotnet/winforms#2389]
* .NET Core Version: (e.g. 3.0 Preview1, or daily build number, use `dotnet --info`)
Runtime Environment: Version: 3.1.0-preview2.19525.6
 
* Have you experienced this same bug with .NET Framework?: No

**Problem description:**
Use this code
```
var colorEditor = TypeDescriptor.GetEditor(typeof(System.Drawing.Color), 
typeof(UITypeEditor));
```
In .net framework, return a ColorEditor instance
In .net core, return null.

**Actual behavior:** 
return null.

**Expected behavior:**
return a ColorEditor instance

**Minimal repro:**
create a dot net core winform project, add this code then get result.

**Comment by vladimir-krestov** [comment]
We probably added the editor definition in later versions of .Net Core 3.1 preview.
At the moment, it works in **master** branch.
![image](https://user-images.githubusercontent.com/49272759/69133033-407da580-0ac6-11ea-99d6-49c08c2d45b4.png)

**Comment by RussKie** [comment]
The `ColorEditor` was hooked up in a1817f9c419bc75f39b36cacfdbea56bddba80ed, so it will be in 3.1 RTM.

**Comment by TempeBrennan** [comment]
> The `ColorEditor` was hooked up in [a1817f9](https://github.com/dotnet/winforms/commit/a1817f9c419bc75f39b36cacfdbea56bddba80ed), so it will be in 3.1 RTM.

Thanks a lot. I will wait 3.1RTM release.

**Comment by TempeBrennan** [comment]
I can see this issue is already fixed in dot net core 3.1. Thanks a lot!

**Comment by RussKie** [comment]
Great! Thank you for the follow up!
